src/agents/pnp/MpiProxy.c

```diff
@@ -62,7 +62,15 @@ int CallMpiSet(const char* componentName, const char* propertyName, const MPI_JS
 
     snprintf(g_mpiCall, sizeof(g_mpiCall), g_mpiCallTemplate, "MpiSet", componentName, propertyName);
 
-    result = MpiSet(g_mpiHandle, componentName, propertyName, payload, payloadSizeBytes);
+    if (IsValidMimObjectPayload(payload, payloadSizeBytes, (GetLog())))

```

Comment:
Are the additional `()` necessary around `GetLog()`?

Reply:
Nope!